torch/_higher_order_ops/cond.py

```diff
@@ -392,22 +396,28 @@ def backward(ctx, *flat_grads):
 
 @cond_op.py_impl(DispatchKey.Autograd)
 def cond_autograd(pred, true_fn, false_fn, operands):
+    suppress_pending = contextlib.nullcontext()
+    if (fake_mode := detect_fake_mode()) and (shape_env := fake_mode.shape_env):
+        shape_env.pending_fresh_unbacked_symbols.clear()
```

Comment:
Clears so we don't have to worry about fresh symbols created in parent graph calls, leaking into cond subgraph analysis